lib/rules/function-max-empty-lines/__tests__/index.js

```diff
@@ -48,50 +49,79 @@ testRule(rule, {
   ],
 
   reject: [
+    {
+      code: "a { transform: translate(/*c*/\n\n\n1,1); }",
+      fixed: "a { transform: translate(/*c*/\n1,1); }",
+      message: messages.expected(0),
+      line: 1,
+      column: 31
+    },
+    {
+      code: "a { transform: /*c*/translate(\n\n\n1,1); }",
+      fixed: "a { transform: /*c*/translate(\n1,1); }",
+      message: messages.expected(0),
+      line: 1,
+      column: 31
+    },
     {
       code: "a { transform: translate(\n\n1\n,\n1\n); }",
+      fixed: "a { transform: translate(\n1\n,\n1\n); }",
+      message: messages.expected(0),
+      line: 1,
+      column: 26
+    },
+    {
+      code: "a { transform: translate(\n\n1\n,\n1\n\n); }",
```

Comment:
I should have made it a little clearer.
In the next case, the fixed css becomes invalid.

```css
a { transform: translate(\n\n1\n\n,\n1\n\n); }
```
->
```css
a { transform: translate(\n1\n,\n1\n\n; }
```

Reply:
Hi, thank you for the example, it was very helpful! I have updated the PR accordingly